plugins/times-of-day/frontend/public/stylesheets/main.scss

```diff
@@ -0,0 +1,28 @@
+@use "sass:map";
+@use "../../../../../frontend/express/public/stylesheets/styles//base/typography-variables" as typography;
+@use "../../../../../frontend/express/public/stylesheets/styles//base/colors" as colors;
+
+.times-of-day {
+
+    &__scatter-chart-tooltip {
+        color: map.get(colors.$colors, "cool-gray-100");
+    }
+
+    &__scatter-chart-tooltip-text {
+        font-size: typography.$text-sm-size;
+        line-height: typography.$text-button-line-height;
+    }
+
+    &__scatter-chart-tooltip-total-users-value {
+        font-weight: typography.$font-weight-primary;
+        font-size: typography.$text-lg-size;
+        line-height: typography.$heading-5-line-height;
+    }
```

Comment:
we also have classes - **text-small** and **text-big**
maybe we can use them instead of creating new classes?

Reply:
I tried to re-use them but unfortunately the text property values did not match. I am sticking to design specifications first as that's how it willbe reviewed. Lack of design consistency is the culprit here I am afraid.